### PR TITLE
[Develop] Feature/import accounts

### PR DESCRIPTION
**Pull request by samuelloza**

### CODE REVIEW

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/ClassicImportAccountPage.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    @Override
+    public void clickImportElement() {
+        GuiInteractioner.clickWebElement(importButton);
+    }
+
+    /**
+     * [SL] This method contain the first steps of the importation.
+     */
+    @Override
+    public void chooseDataStep(final String fieldEmail, final String fileName, final String option) {
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/ClassicImportAccountPage.java:37]: This method should return another diferent Step not the same; when you click "Next" button, where are you diven to?

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    /**
+     * Gets all data of the table Bulk Data Load Job.
+     *
+     * @return a map with the all data of the table
+     */
+    public Map<String, String> bulkDataLoadJobDetail() {
+        Map<String, String> map = new HashMap<>();
+        List<WebElement> headerList = getDriver().findElements(HEADER_TABLE);
+        List<WebElement> contendList = getDriver().findElements(CONTEND_TABLE);
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java]: Please change contend for content

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+        for (int i = 0; i < headerList.size(); i++) {
+            String key = headerList.get(i).getText();
+            String value = contendList.get(i).getText();
+            map.put(key, value);
+        }
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java]: You can reduce this to only one line:   ----->     map.put(headerList.get(i).getText(), contendList.get(i).getText())

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    /**
+     * [SL] Verifies if the download file contains any error or status false.
+     *
+     * @return if found false or any error
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    public boolean downloadedFileNotContainFalse() throws IOException, InterruptedException {
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java:64]: Maybe this verification is not the responsability of the Page Object, I think that you have to create an util to handle CSV files.

And plaese, you should change all "contend" for "content", it is not a valid English word.

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+        return true;
+    }
+
+    /**
+     * [SL] Gets a list with the id of all accounts imported.
+     *
+     * @return a list
+     * @throws IOException
+     */
+    public List<String> accountIdList() throws IOException {
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java]: The same for this, I think it is not the responsability of the Page Object

File: salesforce-gui/src/main/java/org/fundacionjala/salesforce/utils/CSVUtil.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * [SL] This class read a csv file.
+ */
+public final class CSVUtil {
```
Comment (MirkoFer-622) [inline salesforce-gui/src/main/java/org/fundacionjala/salesforce/utils/CSVUtil.java]: MAybe this class should be named CSVReader

File: salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/hooks/AccountHooks.java [hunk 7]
```diff
@@ -35,4 +34,15 @@ public void deleteAccount() {
...
         WebDriverManager.getInstance().quit();
     }
+
+    /**
+     * [SL] Hook that delete an Account saved in Context via API.
+     */
+    @After(value = "@deleteAccounts", order = 1)
+    public void deleteAccounts() {
+        for (String accountId : context.getAccountIdList()) {
+            System.out.println(accountId.replace("\"", ""));
```
Comment (MirkoFer-622) [inline salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/hooks/AccountHooks.java]: Avoid printing lines in develop code.

File: salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/stepdefs/ImportAccountStep.java [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+    public ImportAccountStep(final Context sharedContext) {
+        context = sharedContext;
+    }
+
+    /**
+     * Clicks in the Import button.
+     */
+    @When("I go to Import section in Accounts page")
+    public void goToImportSectionInAccountsPage() {
+    }
```
Comment (MirkoFer-622) [inline salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/stepdefs/ImportAccountStep.java:45]: Delete this step if you are not using it

**Review by MirkoFer-622 — CHANGES_REQUESTED** [review]
Take account of the commentaries I left, please.

File: salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/hooks/AccountHooks.java (hunk 7 above)
Comment (samuelloza) [reply salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/hooks/AccountHooks.java]: Sorry

**Comment by samuelloza** [comment]
@MirkoFer-622 I complete the task, could you review them please?

This PR is broken because need a PR of core

**Review by MirkoFer-622 — APPROVED** [review]

**Pull request merged**